### ISSUE TITLE
Theme not applied to uploaded and renamed files

### ISSUE THREAD

**Issue by fcturner** [owncloud/core#8511]
Hello,

I am using ownCloud 6.0.3 with a theme which edits the following in the files app:
- Give file an icon based on file extension and disable preview to always show my themed icons
- Set real date (21-03-2014 17:21) in Edited column instead of relative date (2 hours ago)

Works like a charm.
However, whenever I upload or rename a file, the newly added or renamed file does not get the modifications of my theme (root\MyTheme\apps\files\parts.list.php)
but reads from the original parts.list.php (root\apps\files\parts.list.php).

In other words; theming is not supported for uploaded or name changed files. It does work when I reload the page. 

Posted here following advice to do so (http://forum.owncloud.org/viewtopic.php?f=26&t=21241)

**Comment by jancborchardt** [comment]
Closing because of inactivity.

**Comment by fcturner** [comment]
It does work now, in another way. I changed the javascript files, instead of the php files. 
Makes more sense too. Thanks anyway!

**Comment by jancborchardt** [comment]
@fcturner good to hear! :)
